lib/operations/execute_operation.js

```diff
@@ -29,6 +29,25 @@ function executeOperation(topology, operation, callback) {
     throw new TypeError('This method requires a valid operation instance');
   }
 
+  if (
+    topology.description != null &&
+    !operation.hasAspect(Aspect.SKIP_SESSION) &&
+    topology.shouldCheckForSessionSupport()
+  ) {
+    // TODO: this is only supported for unified topology, the first part of this check
```

Comment:
Will a non-unified topology just skip this, or will it error b/c it does not have `shouldCheckForSessionSupport`?

Reply:
it will just skip because of the `topology.description` check above. I could probably make a function that has a more descriptive name..

Reply:
yeah, it is not clear that `topology.description` only exists on the unified topology. Maybe a helper `isUnifiedTopology` could be used here.

Reply:
done